### ISSUE TITLE
Add support for offset coordinates

### ISSUE THREAD

**Issue by astrofrog** [astrofrog/wcsaxes#8]

**Comment by gully** [comment]
Just want to bump this feature request, I think it would have loads of use.  The application I have in mind is a "postage stamp" application that cares about the relative position on the sky, not absolute.  

Is there a work-around?  Could there be some way to join this with the `Cutout2D` class in `astropy.nddata.utils`?

<img width="728" alt="screen shot 2016-08-18 at 7 52 06 pm" src="https://cloud.githubusercontent.com/assets/860227/17772786/4911c484-657d-11e6-8390-d4a729a26547.png">

**Comment by astrofrog** [comment]
So it turns out this is actually now somewhat possible with the offset coordinate frames present in Astropy but I need to document it. Could you try and see if you can use a SkyOffsetFrame as an overlay frame in WCSAxes?

http://docs.astropy.org/en/stable/api/astropy.coordinates.SkyOffsetFrame.html#astropy.coordinates.SkyOffsetFrame

**Comment by gully** [comment]
Yes, I am happy to try this.  It'll be a little while until I get back to the project that needed this though.

**Comment by Miranda-J** [comment]
I'm trying to do the same thing, but have no idea how the SkyOffsetFrame could be used to help. I'm relatively new to this.
